docs/layers/terrain-layer.md

```diff
@@ -0,0 +1,171 @@
+<!-- INJECT:"TerrainLayerDemo" -->
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/@deck.gl/geo--layers-lightgrey.svg?style=flat-square" alt="@deck.gl/geo-layers" />
+  <img src="https://img.shields.io/badge/lighting-yes-blue.svg?style=flat-square" alt="lighting" />
+</p>
+
+# TerrainLayer
+
+This TerrainLayer reconstructs mesh surfaces from height map images, e.g. [Mapzen Terrain Tiles](https://github.com/tilezen/joerd/blob/master/docs/formats.md), which encodes elevation into R,G,B values.
+
+When `terrainImage` is supplied with a templated tile server URL, e.g. URL containing "{x}", it renders terrain tiles globally using the TileLayer and SimpleMeshLayer. Otherwise, supply an absolute URL to `terrainImage` and a `bounds` prop to render a any terrain image (e.g. non-Slippy map tiles) with a SimpleMeshLayer.
+
+```js
+import DeckGL from '@deck.gl/react';
+import {TerrainLayer} from '@deck.gl/geo-layers';
+
+export const App = ({viewport}) => {
+
+  const layer = new TerrainLayer({
+    minZoom: 0,
+    maxZoom: 23,
+    strategy: 'no-overlap',
+    elevationDecoder: {
+      rScaler: 256,
+      gScaler: 1,
+      bScaler: 1/256,
+      offset: -32768
+    },
+    terrainImage: 'https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png',
+    surfaceImage: 'https://wms.chartbundle.com/tms/1.0.0/sec/{z}/{x}/{y}.png?origin=nw'
+  });
+  return <DeckGL {...viewport} layers={[layer]} />;
+};
+```
+
+## Installation
+
+To install the dependencies from NPM:
+
+```bash
+npm install deck.gl
+# or
+npm install @deck.gl/core @deck.gl/layers @deck.gl/geo-layers
+```
+
+```js
+import {TerrainLayer} from '@deck.gl/geo-layers';
+new TerrainLayer({});
+```
+
+To use pre-bundled scripts:
+
+```html
+<script src="https://unpkg.com/deck.gl@^8.0.0/dist.min.js"></script>
+<!-- or -->
+<script src="https://unpkg.com/@deck.gl/core@^8.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/layers@^8.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/geo-layers@^8.0.0/dist.min.js"></script>
+```
+
+```js
+new deck.TerrainLayer({});
+```
+
+## Properties
+
+When in Tiled Mode, inherits from all [TileLayer](/docs/api-reference/tile-layer.md) properties. Forwards `wireframe` and `color` (via `getColor`) properties to [SimpleMeshLayer](/docs/api-reference/simple-mesh-layer.md).
```

Comment:
Properly document `wireframe` and `color` in the properties list. `color` is not technically "forwarded" because `getColor` receives a function.

Reply:
Where is the properties list?

Reply:
Under `### Mesh Options`